web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -1,5 +1,11 @@
 const AWS = require('aws-sdk');
 const createApplicationContext = require('../../../src/applicationContext');
+const {
+  migrateItems: migration0024,
+} = require('./migrations/0024-docket-entry-docket-number-served-parties-code');
+const {
+  migrateItems: migration0025,
+} = require('./migrations/0025-add-contacts-to-petitioners-array');
 
 const {
   migrateItems: migration0024,
```

Comment:
@rachelschneiderman -- FYI, you'll need to remove one of these; I got it going manually for tonight.

Reply:
Thank you! I'll do that now so we don't forget.